dea_conflux/__main__.py

```diff
@@ -427,41 +427,46 @@ def get_ids(product, expressions, verbose, s3):
     help='Visibility timeout in seconds',
     default=18 * 60)
 @click.option(
-    '--deadletter', default=None,
-    help='Name of deadletter queue')
-@click.option(
-    '--retentionperiod', type=int,
+    '--retentionperiod', type=str,
```

Comment:
str?

Reply:
also, please use `retention-period`

Reply:
haha, yes. String format. I run it by int, but AWS said `I needs a string`.

Reply:
fair enough!

Reply:
perhaps better to use `int` here and then convert to string later, that way we get the good typing